recipes/bzip2/all/conanfile.py

```diff
@@ -1,6 +1,7 @@
-from conan import ConanFile
+from conan import ConanFile, conan_version
```

Comment:
This should not be used

```suggestion
from conan import ConanFile
```

Reply:
You got it!